### ISSUE TITLE
A console window is always being popped up

### ISSUE THREAD

**Issue by lou-xia** [HJLebbink/asm-dude#161]
A console window is always being popped up, and it is useless to close it or use `Ctrl+c`. 
It is annoying!!! 
I wonder how to close it, or wether it is a bug.

![image](https://github.com/user-attachments/assets/a2912717-b200-4735-bf66-a4a29692c546)

![image](https://github.com/user-attachments/assets/0c786c64-c51d-4c94-ba8a-825faed30629)

**Comment by RayKoopa** [comment]
If you close this window, you kill off the language server, breaking most of the extension's functionality.

The console should not pop up in Release builds according to the [source](https://github.com/HJLebbink/asm-dude/blob/9e1ab93e4aefa16de7e93ded5ed684f7fd20984d/VS/CSHARP/asm-dude2-ls/Worker.cs#L25) so I wonder if this was built with DEBUG?

**Comment by lou-xia** [comment]
> The console should not pop up in Release builds according to the [source](https://github.com/HJLebbink/asm-dude/blob/9e1ab93e4aefa16de7e93ded5ed684f7fd20984d/VS/CSHARP/asm-dude2-ls/Worker.cs#L25) so I wonder if this was built with DEBUG?

Actually, I download it from here: https://marketplace.visualstudio.com/items?itemName=Henk-JanLebbink.AsmDude2&ssr=false#review-details

And you can find others reported this bug.

**Comment by RayKoopa** [comment]
Yes, I was wondering if the released extension on the marketplace was was compiled in DEBUG. Maybe the developer can chime in on this.
